controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java

```diff
@@ -67,8 +68,8 @@ public Task get(@PathParam("id") Long id) {
    */
   @GET
   @Produces(MediaType.APPLICATION_JSON)
-  public List<Task> findTasks(@QueryParam("clusterName") Optional<String> clusterName,
-                              @QueryParam("state") Optional<TaskState> state) {
+  public List<TaskInternal> findTasks(@QueryParam("clusterName") Optional<String> clusterName,
```

Comment:
if task internal will be exposed by http, maybe give it a better name because it will be seen by end user